### PR TITLE
 python3-stem: update to 1.8.0 

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/python-stem/template [hunk 1]
```diff
@@ -3,28 +3,9 @@ pkgname=python-stem
...
+build_style=meta
+short_desc="Python2 controller library for Tor(removed package)"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="LGPL-3"
+license="LGPL-3.0-only"
 homepage="https://stem.torproject.org/"
-changelog="https://stem.torproject.org/change_log.html#version-1-7"
-distfiles="${PYPI_SITE}/s/stem/stem-${version}.tar.gz"
-checksum=c9eaf3116cb60c15995cbd3dec3a5cbc50e9bb6e062c4d6d42201e566f498ca2
 alternatives="stem:tor-prompt:/usr/bin/tor-prompt2"
```
Comment (xtraeme) [inline srcpkgs/python-stem/template]: This line shouldn't be there

File: srcpkgs/python-stem/template [hunk 2]
```diff
@@ -3,28 +3,9 @@ pkgname=python-stem
 version=1.7.1
 revision=2
```
Comment (xtraeme) [inline srcpkgs/python-stem/template]: Revbump

Comment (ndowens) [reply srcpkgs/python-stem/template]: Why would we revbump something that is meta(for removal)?

Comment (xtraeme) [reply srcpkgs/python-stem/template]: Because the existing pkg in the xbps repo won't be updated

File: srcpkgs/python3-stem/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-stem'
+pkgname=python3-stem
+version=1.8.0
+revision=1
+archs=noarch
+wrksrc="stem-${version}"
+build_style=python3-module
+pycompile_module="stem"
```
Comment (xtraeme) [inline srcpkgs/python3-stem/template]: This one should be auto detected by the hook

Comment (ndowens) [reply srcpkgs/python3-stem/template]: Yep, I think this was done possibly before I found out about not adding and start removing pycompile line.

**Comment by Chocimier** [comment]
> Revbump

@xtraeme, could we stop furtively remove packages from user systems with empty packages, and go for something explicit like adding this operation to xbps or [remove-packages](https://github.com/void-linux/void-packages/pull/17578) + [unindex](https://github.com/void-linux/xbps-bulk/pull/2) ?

**Pull request closed without merging**